src/hooks/useKeyboardControls.ts

```diff
@@ -22,9 +23,11 @@ interface UseKeyboardControlsProps {
   isSeatTaken: (row: number, column: number) => boolean;
   takeSeat: (row: number | null, column: number | null) => void;
   enterSelectedRoom: () => void;
-  onMove?: () => void;
+  onMove?: () => void; // TODO: this doesnt seem to be needed anymore, can we remove it?
 }
 
+// TODO: use e.preventDefault() or return false or similar in the keyboard handlers (check mousetrap docs) so we don't scroll on arrow key presses
+// TODO: we may also need some kind of 'centre me on the map' logic somewhere when we do this too? For large zoomed maps.
 // TODO: implement bindRef using useRef() or similar (in Camp?) then remove all withGlobalBind (default is false)
```

Comment:
- [ ] [TODO]: implement `bindRef` using `useRef()` or similar (in `Camp`?) then remove all `withGlobalBind` (default is `false`)

Note: there is an existing bug (before this PR) related to this, but it shouldn't actually impact us until (if) we switch to restricting the keypresses to only be listened for within a certain component.

Reply:
see https://github.com/sparkletown/internal-sparkle-issues/issues/5